README.md

```diff
@@ -41,15 +41,15 @@ First add the latest version of Elasticsearch-Evolution spring boot starter as a
 <dependency>
     <groupId>com.senacor.elasticsearch.evolution</groupId>
     <artifactId>spring-boot-starter-elasticsearch-evolution</artifactId>
-    <version>0.2.1</version>
+    <version>0.3.0</version>
 </dependency>
 ```
 
-Elasticsearch-Evolution uses internally Elastics RestHighLevelClient and requires at minimum version 6.6.0. Spring boot uses a older version, so update it in your pom.xml:
+Elasticsearch-Evolution uses internally Elastics RestHighLevelClient and requires at minimum version 7.5.0. Spring boot uses a older version, so update it in your pom.xml:
 
 ```xml
 <properties>
-    <elasticsearch.version>6.8.6</elasticsearch.version>
+    <elasticsearch.version>7.5.0</elasticsearch.version>
```

Comment:
better use the latest patch version `7.5.2`

Reply:
done